README.FreeBSD

```diff
@@ -11,152 +11,112 @@ In the end the cluster would be running bhyve on RBD disk that are stored in
+   Speed is not impressive, but is does work.
+ - rbd-ggate available to create a Ceph rdb backed device. 
+   rbd-ggate was submitted by Mykola Golub
+   That works rather simple:
+     "build a cluster" 
+     dd if=/dev/zero of=/tmp/image.1G bs=1M count=1k
+     rdb --pool rbd import /tmp/image.1G newdevice
+     rbd-gate map rbd/newdevice
+     newfs /dev/ggate
+   And there is a 1G disk stored on a ceph-cluster.
 
 Other improvements:
+ - bugs in the init-ceph code (needed for rc.d) are being fixed
+ - RBD and rados work,
+ - compatability code was written so FreeBSD and Linuxs daemons can 
+   operate together.
+ - More of the awkward dependancies on Linux-isms are deleted only 
+      But /bin/bash is there to stay.
 
-	* A new ceph-devel update will be submitted in April
-
-  -	Ceph-Fuse works, allowing to mount a CephFS on a FreeBSD system and do
-	some work on it. 
-  -	Ceph-disk prepare and activate work for FileStore on ZFS, allowing 
-	easy creation of OSDs.
-  -	RBD is actually buildable and can be used to manage RADOS BLOCK 
-	DEVICEs.
-  -	Most of the awkward dependencies on Linux-isms are deleted only 
-	/bin/bash is there to stay.
-
-Getting the FreeBSD work on Ceph:
-=================================
-
-	pkg install net/ceph-devel
-
-Or:
-	cd "place to work on this"
-	git clone https://github.com/wjwithagen/ceph.git
-	cd ceph
-	git checkout wip.FreeBSD
+Expected real soon:
+  The next official release of Ceph is called Luminous (v12.1.0).
+  As soon as that is available, a port will be made available.
 
-Building Ceph
-=============
-  - 	Go and start building
-	  ./do_freebsd.sh
-	
 Parts not (yet) included:
 =========================
-
+      
  - KRBD
-	 Kernel Rados Block Devices is implemented in the Linux kernel
-	 And perhaps ggated could be used as a template since it does some of 
-	 the same, other than just between 2 disks. And it has a userspace 
-	 counterpart.
+   Use rbd-ggate
  - BlueStore.
-	 FreeBSD and Linux have different AIO API, and that needs to be made 
-	 compatible Next to that is there discussion in FreeBSD about 
-	 aio_cancel not working for all devicetypes
- - CephFS as native filesystem
-	 (Ceph-fuse does work.)
-	 Cython tries to access an internal field in dirent which does not 
-	 compile
+   FreeBSD and Linux have different AIO API, and that needs to be made 
+   compatible. Next to that is there discussion in FreeBSD about 
+   aio_cancel not working for all devicetypes
+
+   
 Build Prerequisites
 ===================
 
-	Compiling and building Ceph is tested on 12-CURRENT, but I guess/expect 
-	11-RELEASE will also work. And Clang is at 3.8.0.
-	It uses the CLANG toolset that is available, 3.7 is no longer tested, 
-	but was working when that was with 11-CURRENT. 
-	Clang 3.4 (on 10.2-STABLE) does not have all required capabilites to 
-	compile everything
+   Compiling and building Ceph is tested on 12-CURRENTclang 4.0.0, i
+   but I 11-RELEASE/clang 3.8.0.
```

Comment:
@tchaikov 

That paragraph was a mess in the wip. Took a fresh one from the FreeBSD reporting.
Should be more legible now.